tests/configs/base.yaml

```
@@ -1,9 +1,16 @@
 model:
   ckpt_path: null
-  encoder_cfg:
-    model_name: "resnet18"
-    in_chans: 3
-    pretrained: False
+  encoder_cfg: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)